python/gink/impl/connection.py

```diff
@@ -27,12 +29,18 @@ def __init__(
             host: Optional[str] = None,
             port: Optional[int] = None,
             name: Optional[str] = None,
-            socket: Optional[Socket] = None,
-    ):
+            socket: Optional[Socket] = None
+            ):
         if socket is None:
             assert host is not None and port is not None
             socket = Socket(AF_INET, SOCK_STREAM)
+            cabundle_path = environ.get("GINK_CABUNDLE")
+            if cabundle_path:
```

Comment:
This should be passed in as an argument rather than read from an environment variable.

Reply:
Additionally, we should ask if the user wants to initiate a secure connection as a boolean, then optionally pass in the the cabundle_path, otherwise using the system default (see `ssl.create_default_context`)